jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/SamDebugSupport.kt

```diff
@@ -16,24 +17,25 @@ interface SamDebugSupport {
     val debuggerAttachTimeoutMs: Long
         get() = 60000L
 
-    fun patchCommandLine(debugPort: Int, state: SamRunningState, commandLine: GeneralCommandLine) {
-        commandLine.withParameters("--debug-port")
-            .withParameters(debugPort.toString())
+    fun patchCommandLine(debugPorts: List<Int>, commandLine: GeneralCommandLine) {
+        commandLine.withParameters("--debug-port").withParameters(debugPorts.first().toString())
```

Comment:
Just picking the first one?

Reply:
Ya, if you really care override the method :D

Reply:
Ideally we could do:
```
for each item in debug port:
define the param
```

but not sure what sam does if you define it more than once.

Reply:
Eh, ill do that change since number of ports is gated by the support version anyway